test/Elastic.Apm.Tests/HelpersTests/AgentTimerTests.cs

```diff
@@ -319,33 +319,38 @@ public void CompleteTaskSuccessfully()
 
 			public void VerifyTryAwaitCompletedSuccessfully(Task tryAwaitOrTimeoutTask, Task delayTask)
 			{
-				tryAwaitOrTimeoutTask.IsCompletedSuccessfully.Should().BeTrue();
```

Comment:
Unfortunately there is no `Task.IsCompletedSuccessfully` on `net461`.

Reply:
I would add `IsCompletedSuccessfully()` extension method - https://github.com/elastic/apm-agent-dotnet/pull/517/commits/8e3f8afc2171bffc97b4b583736b4459d3d998f7 and https://github.com/elastic/apm-agent-dotnet/pull/517/commits/0826ed36910b94d3fb2349717142812068eb49c9